src/main/java/model/Car.java

```diff
@@ -0,0 +1,101 @@
+package model;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Arrays.*;
+import static java.util.stream.Collectors.toList;
+
+public class Car {
+    public static final int START_SCORE = 1;
+    private int position;
```

Comment:
상수와 인스턴스 변수의 구별이 쉽도록 한줄 띄어쓰면 좋을 것 같아요. 

```suggestion
    public static final int START_SCORE = 1;
    
    private int position;
```